src/postman/operation.ts

```diff
@@ -2,24 +2,40 @@ import {
   HttpParamStyles,
   HttpSecurityScheme,
   IHttpHeaderParam,
+  IHttpOperation,
   IHttpOperationRequestBody,
   IHttpParam,
   IHttpPathParam,
   IHttpQueryParam,
   IMediaTypeContent,
 } from '@stoplight/types';
-import { Collection, Item, ItemGroup, RequestAuth, Url } from 'postman-collection';
+import { Collection, CollectionDefinition, Item, ItemGroup, RequestAuth, Url } from 'postman-collection';
 import { transformRequest } from './transformers/request';
 import { transformResponse } from './transformers/response';
 import {
   isPostmanSecuritySchemeEqual,
+  PostmanSecurityScheme,
   transformSecurityScheme,
   transformSecuritySchemes,
 } from './transformers/securityScheme';
 import { transformServer } from './transformers/server';
 import { PostmanCollectionHttpOperationTransformer } from './types';
 import { transformDescriptionDefinition, traverseItemsAndGroups } from './util';
 
+export const transformPostmanCollectionOperations = (document: CollectionDefinition): IHttpOperation[] => {
+  const collection = new Collection(document);
+  const resolvedCollection = new Collection(collection.toObjectResolved({ variables: collection.variables }, []));
+
+  const securitySchemes = transformSecuritySchemes(collection);
```

Comment:
Ok I like this better, transforming once forever and then passing it around 👍

Reply:
Yeah, this is the pattern we can efficiently use when parsing all operations.